### DISCUSSION TITLE
Crankshaft interrupting boot up video

### DISCUSSION THREAD

**Discussion by GHNeko** [CrankshaftForSteam/crankshaft#20] [General]
There are a few of us who have custom boot up videos, mine in particular is a Steam Deck version of the Dreamcast Boot Up animation with sound and moving laser sounds and all that jazz. 

But upon crankshaft starting up during the steam client boot, it ends the boot video prematurely. 

Wanted to know if there was any way to stop this from happening or delaying cranksaft from cutting the video short.

**Comment by ruineka** [comment]
I also would like to know this, I've ran into the same problem.

**Comment by dotaxis** [comment]
You can add a delay to the start of the service, which is the best I've been able to figure out. 

Edit the following file, either via SSH or desktop mode:
`/home/deck/.config/systemd/user/crankshaft.service`

Under the line that reads `[Service]` add this on its own line:
`ExecStartPre=/bin/sleep 15`

That will delay the start of Crankshaft by 15 seconds, but it'll reload the UI when it does launch and it's a bit janky
